2016/Call.js

```diff
@@ -1,19 +1,13 @@
 'use strict';
 
 var GetIntrinsic = require('../GetIntrinsic');
+var callBound = require('../helpers/callBound');
 
-var $TypeError = GetIntrinsic('%TypeError%');
-
-var inspect = require('object-inspect');
-
-var IsCallable = require('./IsCallable');
+var $apply = GetIntrinsic('%Reflect.apply%', true) || callBound('%Function.prototype.apply%');
 
 // https://www.ecma-international.org/ecma-262/6.0/#sec-call
 
 module.exports = function Call(F, V) {
 	var args = arguments.length > 2 ? arguments[2] : [];
-	if (!IsCallable(F)) {
-		throw new $TypeError(inspect(F) + ' is not a function');
-	}
```

Comment:
i kind of prefer the explicit error here; what do you think?

Reply:
Well, `Reflect.apply(…)` takes care of that for us.

Reply:
Although in general I prefer explicit error messages, in this case it seems like Reflect.apply's  error message is good.